Source/WebCore/Modules/mediastream/MediaStreamTrack.h

```diff
@@ -65,6 +65,7 @@ class MediaStreamTrack
 {
     WTF_MAKE_TZONE_OR_ISO_ALLOCATED(MediaStreamTrack);
 public:
+    USING_CAN_MAKE_WEAKPTR(EventTarget);
```

Comment:
This pull request includes some examples with a blank line between `USING_CAN_MAKE_WEAKPTR`  and `ref`/`deref` and others without. I would prefer consistency, although I don’t have preference for which style. Also, in some places `USING_CAN_MAKE_WEAKPTR` is after the `ref`/`deref` functions, another unnecessary inconsistency.
```suggestion
    USING_CAN_MAKE_WEAKPTR(EventTarget);

```